Sources/VaporSecurityHeaders/Configurations/ContentSecurityPolicyConfiguration.swift

```diff
@@ -1,11 +1,12 @@
 import Vapor
+import Foundation
 
 public struct ContentSecurityPolicyConfiguration: SecurityHeaderConfiguration {
-
+    
     private let value: String
 
-    public init(value: String) {
```

Comment:
I think we should leave this constructor as an override option for those that want it (also makes it a non-breaking change!)